gp-includes/default-filters.php

```diff
@@ -3,6 +3,6 @@
  * Filters and actions assigned by default
  */
 
-add_action( 'gp_head', 'wp_enqueue_scripts' );
-add_action( 'gp_head', 'gp_print_styles' );
-add_action( 'gp_head', 'gp_print_scripts' );
\ No newline at end of file
+//add_action( 'gp_head', 'wp_enqueue_scripts' );
```

Comment:
I think that is not allowed also for backcompatibility.
Maybe add a filter that enable to disable this hook is the best way